draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -433,33 +434,31 @@ retransmits RTP to receivers via unicast. QUIC can be used between the
 *Retransmission Source* and the receivers.
 
 Note-Topo-PtM-Trn-Translator:
-: Supports unicast paths between RTP sources and translators.
+: Supported for IP unicast paths between RTP sources and translators.
 
 Note-Topo-Mixer:
-: Supports unicast paths between RTP senders and mixers.
+: Supported for IP unicast paths between RTP senders and mixers.
 
 Note-Warn:
-: Quote from {{?RFC7667}}: *This topology is so problematic and it is so easy to
-get the RTCP processing wrong, that it is NOT RECOMMENDED to implement this
-topology.*
+: Quote from {{?RFC7667}}: *This topology is so problematic and it is so easy to get the RTCP processing wrong, that it is NOT RECOMMENDED to implement this topology.*
 
-# Connection Establishment and ALPN {#alpn}
+# Connection Establishment and Application-Layer Protocol Negotiation {#alpn}
 
-QUIC requires the use of ALPN {{!RFC7301}} tokens during connection setup. RoQ
-uses "roq" as ALPN token in the TLS handshake (see also
+QUIC requires the use of Application-Layer Protocol Negotiation (ALPN) {{!RFC7301}} tokens during connection setup. RoQ
+uses "roq" as the ALPN token, included as part of the TLS ClientHello message (see also
```

Comment:
I think *handshake* is better, because it is used in ClientHello and ServerHello.

Reply:
You're right about that.